### DISCUSSION TITLE
Idea: CLSN Boxes Can be scaled via Helper summoning

### DISCUSSION THREAD

**Discussion by cyliamargatroid** [ikemen-engine/Ikemen-GO#831] [Feature Request] [answered]
```
[State 9400, 1]
type = Helper
trigger1 = time = 0
ID = 1950
pos = -50,0
postype = back
stateno = 1950
helpertype = normal
name = "Assist (Ruby Rose)"
keyctrl = 0
ownpal = 1
size.xscale = .125
size.yscale = .125
; CLSN resizable (Useful on Common Games too)
clsn.xscale = .125 ; Will be set the same size as your CLSN Scaling if ommited
clsn.yscale = .125 ; Will be set the same size as your CLSN Scaling if ommited

**Comment by cyliamargatroid** [comment]
I even tried DIO (Warusaki3's Version) on Interactive MUGEN Patched on IKEMEN GO, but The CLSN Size doesn't seem correct in common.air, but Characters with size.xscale = 1 and size.yscale = 1 works.

Patchable for IKEMEN GO
https://www.youtube.com/watch?v=gcAyOmOXuQQ

**Comment by Windblade-GR01** [comment]
Nice ideas about scalable boxes!
Maybe it could be easy to implement.

About interactive mugen maybe it would be better to use `Attachedchar`
and port the code to the stage itself.

**Comment by potsmugen** [comment, marked as answer]
We already made Clsn scalable with `TransformClsn`.
